<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Telepro-Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(167, 154, 91);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 255, 0);
background-color: rgb(77, 93, 70);
alternate-background-color: rgb(85, 170, 127);
border-top-color: rgb(0, 85, 255);
border-right-color: rgb(85, 170, 255);
border-left-color: rgb(255, 0, 127);
border-left-color: rgb(0, 0, 127);
background-color: rgb(152, 168, 125);</string>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::NoVerticalHeader</enum>
    </property>
    <property name="navigationBarVisible">
     <bool>true</bool>
    </property>
    <property name="dateEditEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>.....</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reserve</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>send sms</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_log">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>sent tcp msg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disconnect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calendarWidget</sender>
   <signal>selectionChanged()</signal>
   <receiver>label_2</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
